app/controllers/concerns/api_error_responses.rb

```diff
@@ -4,6 +4,7 @@ module ApiErrorResponses
   included do
     include ActionController::MimeResponds
     rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+    rescue_from StandardError, with: :error
```

Comment:
Unterdrückt das nicht alle Fehler im Development Mode?
In Production sollten die nicht erscheinen, oder was löst der rescue_from hier?

https://github.com/voc/voctoweb/blob/master/config/environments/production.rb#L13

Ah, das ist für JSON Error Responses?

Reply:
Ja, korrekt es war sehr verwirrend da HTML Fehler zu bekommen, wenn ich eigentlich mit der API spreche. Ich muss aber nochmal ran und mir genauer anschauen warum der Test da failed. Ggf. auch nochmal splitten und zwei PRs draus machen oder so.

Reply:
Ja, da jetzt immer 500er kommen kann die API keine 404 not found  mehr signalisieren. Da brauchen wir mehr Logik, sorry.